Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115_test.py

```diff
@@ -347,3 +350,323 @@ def test_paging_command(mocker):
+    mocker.patch.object(RSANetWitnessv115, "argToBoolean", return_value=True)
+    mocker.patch.object(RSANetWitnessv115, "arg_to_number", return_value=2)
+    mocker.patch.object(client, "get_incident_request", return_value={"id": 1, "status": "Closed", "alertCount": 2})
+    mocker.patch.object(RSANetWitnessv115, "fetch_alerts_related_incident", return_value={"alerts": [{"INC-2"}]})
+
+    res = get_remote_data_command(client, {}, {'close_incident': 0, 'import_alerts': True, 'max_alerts': 1})
+    assert res.mirrored_object == paging_data
+
+
+def test_get_remote_data_command_with_new_xsoar_incident_entries(mocker):
+    """
+        Given:
+        - client with fetch parameters
+        - args with incident attributes
+        - params
+
+        When:
+            running get_remote_data_command.
+
+        Then:
+            Update context of incident if incident has been updated from RSA.
+    """
+    paging_data = {'alertCount': 3, 'alerts': [1, 2, 3], 'id': 'INC-1', 'status': 'New'}
+
+    class GetRemoteDataArgsResponse:
+        def __init__(self) -> dict:
+            self.last_update = 1234567890
+            self.remote_incident_id = 0
+
+    mocker.patch.object(RSANetWitnessv115, "GetRemoteDataArgs", return_value=GetRemoteDataArgsResponse())
+    mocker.patch.object(RSANetWitnessv115, "argToBoolean", return_value=True)
+    mocker.patch.object(RSANetWitnessv115, "arg_to_number", return_value=5)
+    mocker.patch.object(client, "get_incident_request", return_value={"id": "INC-1", "status": "New", "alertCount": 3})
+    mocker.patch.object(RSANetWitnessv115, "fetch_alerts_related_incident", return_value=[1, 2, 3])
+
+    res = get_remote_data_command(client, {}, {'close_incident': 0, 'import_alerts': True, 'max_alerts': 5})
+    assert res.mirrored_object == paging_data
+
+
+def test_get_modified_remote_data_command_from_timestamp(mocker):
+    """
+        Given:
+        - client with fetch parameters
+        - args with incident attributes
+        - params
+
+        When:
+            running get_modified_remote_data_command.
+
+        Then:
+            check the updated incidents list.
+    """
+    paging_data = []
+
+    class GetModifiedRemoteDataArgsResponse:
+        def __init__(self) -> dict:
+            self.last_update = "1694188115"
+
+    mocker.patch.object(RSANetWitnessv115, "GetModifiedRemoteDataArgs", return_value=GetModifiedRemoteDataArgsResponse())
+    mocker.patch.object(client, "get_incident_request", return_value={"id": "INC-1", "status": 1, "alertCount": 3})
+    mocker.patch.object(RSANetWitnessv115, "paging_command", return_value=({}, []))
+    mocker.patch.object(RSANetWitnessv115, "clean_old_inc_context", return_value=False)
+    mocker.patch.object(RSANetWitnessv115, "get_integration_context", return_value={})
+
+    res = get_modified_remote_data_command(client, {}, {"max_fetch": 2, "max_alerts": 2, "max_mirror_time": 0})
+    assert res.modified_incident_ids == paging_data
```

Comment:
Line 1334, the API call is filtering the incident to mirror, see the argument until and since.
So this function will never return an incident that shouldn't be mirror, so we can't test what you suggested

Reply:
Okay, so just add a single incident that should return from the **paging_command**, and assert that it is in the modified ids list.